src/core/server/saved_objects/service/lib/internal_utils.test.ts

```diff
@@ -241,3 +242,35 @@ describe('#rawDocExistsInNamespace', () => {
     });
   });
 });
+
+describe('#isNotFoundFromUnsupportedServer', () => {
+  it('returns true with not found response from unsupported server', () => {
+    const rawResponse = {
+      statusCode: 404,
+      headers: {},
+    };
+
+    const result = isNotFoundFromUnsupportedServer(rawResponse);
+    expect(result).toBeTruthy();
```

Comment:
In each of these tests, probably better to assert `toBe(true)` and `toBe(false)` instead?

It is expected that the returned value is always a boolean, so `toBeTruthy` and `toBeFalsy` could accept invalid responses.

Reply:
done (e.g.https://github.com/elastic/kibana/pull/108749/files#diff-ffa64e1fef5834d68badd4984ef0edccdef2a00c612c28f5754fb84cd4fc3f9cR19)